blog/2022/02/10years.md

```diff
@@ -6,6 +6,10 @@
 @def mintoclevel=2 
 @def maxtoclevel=3
 
+~~~
+<img src="https://user-images.githubusercontent.com/35577566/153906824-4c719e96-496e-473c-9771-9fec177d9b5b.png" width="80%" alt="Why We Use Julia, 10 Years Later">
```

Comment:
I agree it's good to have an imagine, but maybe @cormullion can spruce it up a bit visually :)?

Reply:
Yeah. I don't think the image should have the text, it'll clash with the document title.

Reply:
I added the text so it gives a nice preview on twitter but I can remove it